source/Government.cpp

```diff
@@ -457,7 +457,7 @@ double Government::AttitudeToward(const Government *other) const
 	if(other == this)
 		return 1.;
 
-	return attitudeToward.count(other) ? attitudeToward.at(other) : 0.;
+	return attitudeToward.find(other) != attitudeToward.end() ? attitudeToward.at(other) : 0.;
```

Comment:
Using the find you can only look for the attribute once by storing it in a variable